### DISCUSSION TITLE
Settings for Search

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#105] [Q&A] [answered]
As (offline) search becomes an integral part of Cruiser, we could have more settings to accommodate various user preferences.

And better yet they should be organized on their own screen:

<img src="https://user-images.githubusercontent.com/3484020/228541523-b4f9fd52-ed38-4164-9116-a843c18b8e79.jpg" width="50%"/><img src="https://user-images.githubusercontent.com/3484020/228541529-a993b15d-a550-4647-9ddb-11c086df0884.jpg" width="50%"/>

**Comment by devemux86** [comment, marked as answer]
Settings for search are available in [Cruiser 3.0.16](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by DolomitesTom** [reply]
works fine... question: the distance is that now always meant from the actual gps position or from the center of the map sectioan actually displayed?

**Reply by devemux86** [reply]
GPS location (if available) or map center (if location is not available).
Do you think we need something different?

_The distance setting affects the **offline search in points of interest**._

The **online search** and the **offline search in favorites** have no limits.

**Reply by DolomitesTom** [reply]
> GPS location (if available) or map center (if location is not available).
Do you think we need something different?

That should be enough, I can't think of anything missing at the moment.
